src/components/ens-registration/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import Svg, { Path } from 'react-native-svg';
+import Spinner from '../../Spinner';
+import ButtonPressAnimation from '../../animations/ButtonPressAnimation';
+import {
+  AccentColorProvider,
+  BackgroundProvider,
+  Box,
+  Cover,
+  Text,
+  useForegroundColor,
+} from '@rainbow-me/design-system';
+import { useENSProfileForm, useSelectImageMenu } from '@rainbow-me/hooks';
+import { ImgixImage } from '@rainbow-me/images';
+
+const size = 70;
+
+export default function Avatar({
+  onChangeAvatarUrl,
+}: {
+  onChangeAvatarUrl: (url: string) => void;
+}) {
+  const { values, onBlurField, setDisabled } = useENSProfileForm();
+
+  const [avatarUrl, setAvatarUrl] = useState(values?.avatar);
+  useEffect(() => {
+    setAvatarUrl(values?.avatar);
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const accentColor = useForegroundColor('accent');
+
+  const { ContextMenu, isUploading } = useSelectImageMenu({
+    imagePickerOptions: {
+      cropperCircleOverlay: true,
+      cropping: true,
+    },
+    menuItems: ['library'],
+    onChangeImage: ({ image }) => {
+      setAvatarUrl(image.path);
+      onChangeAvatarUrl(image.path);
+    },
+    onUploadError: () => {
+      onBlurField({ key: 'avatar', value: '' });
+      setAvatarUrl('');
+      setDisabled(false);
+    },
+    onUploading: ({ image }) => {
+      onBlurField({ key: 'avatar', value: image.path });
+      setDisabled(true);
+    },
+    onUploadSuccess: ({ data }) => {
+      onBlurField({ key: 'avatar', value: data.url });
+      setDisabled(false);
+    },
+    uploadToIPFS: true,
+  });
+
+  return (
+    <Box height={{ custom: size }} width={{ custom: size }}>
+      <Cover alignHorizontal="center">
+        <BackgroundProvider color="body">
+          {({ backgroundColor }) => (
+            <Svg height="32" style={{ top: -6 }} viewBox="0 0 96 29" width="96">
```

Comment:
could we move this one to `src/components/icons/svg/` just to have all svgs there

Reply:
ya!!

Reply:
actually, thinking about this, does it make sense to move it there? this SVG is not an icon

Reply:
maybe under src/components/svg/ ?

Reply:
sounds good!